### DISCUSSION TITLE
Need Help with generating Gene level Copy Number Ratio

### DISCUSSION THREAD

**Discussion by MaxSimonNm** [etal/cnvkit#1016] [General]
Upon following the steps, i eventually get the call.cns.
When i open the call.cns file, i get the copy numbers for each segment.. which have multiple genes in them.

Is it possible somehow to get individual gene level copy number ratios?

**Comment by Anegin24** [comment]
you can modify the result using dplyr in R

**Comment by etal** [comment]
Use the `genemetrics` command with .cnr and/or .cns to get copy ratios and other stats grouped by gene.
